aws/container-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -61,6 +61,9 @@ systemd:
           --lock-file=/var/run/lock/kubelet.lock \
           --network-plugin=cni \
           --node-labels=node.kubernetes.io/node \
+          %{ for label in split(";", node_labels) }
```

Comment:
Since `var.node_labels` is of type `list`, can't we get rid of the `join/split` here?


(Sorry if useless

Reply:
that was my initial take too, but tf only takes strings as template vars.
(https://github.com/hashicorp/terraform/issues/9368 , etc..)